bin/api_diff.ml

```diff
@@ -6,7 +8,13 @@ let run (`Ref_cmi reference) (`Current_cmi current) =
       ~current:current.cmi_sign
   in
   if differs = [] then Printf.printf "API unchanged!\n"
-  else Printf.printf "API changed!\n"
+  else
+    match differs with
```

Comment:
Probably good to rename `differs` which was named that way when it was a boolean to `diff`.